src/simplification/Simplify.ts

```diff
@@ -13,13 +13,17 @@ let anonymCounter = 1;
  */
 export function simplifyRecursive(schema : Schema | boolean) : CommonModel[] {
   let models : CommonModel[] = [];
-  let types = simplifyTypes(schema);
   let simplifiedModel = simplify(schema);
-  //Only if it contains object and is the only type
-  if(types !== undefined && typeof schema !== "boolean" && types.includes("object") && simplifiedModel[0].properties !== undefined){
-    let rootModel = new CommonModel();
-    rootModel.$ref = schema.$id;
-    models[0] = rootModel;
+  if(simplifiedModel.length > 0){
```

Comment:
`if (simplifiedModel.length)` is enough 😄  As I remember you cannot have length of Array less than 0.

Reply:
True, I like the explicit syntax though 😄